cuth

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
```

Comment:
Nice! I have to do this too, as on my system `perl` is located in `/usr/local/bin`.